hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -126,11 +129,11 @@ public class HoodieWriteConfig extends HoodieConfig {
       .withDocumentation("Payload class used. Override this, if you like to roll your own merge logic, when upserting/inserting. "
           + "This will render any value set for PRECOMBINE_FIELD_OPT_VAL in-effective");
 
-  public static final ConfigProperty<String> MERGE_CLASS_NAME = ConfigProperty
-      .key("hoodie.datasource.write.merge.class")
-      .defaultValue(HoodieAvroRecordMerge.class.getName())
-      .withDocumentation("Merge class provide stateless component interface for merging records, and support various HoodieRecord "
-          + "types, such as Spark records or Flink records.");
+  public static final ConfigProperty<String> MERGER_STRATEGY = ConfigProperty
+      .key("hoodie.datasource.write.merge.strategy")
+      .defaultValue(HoodieAvroRecordMerger.class.getName())
+      .withDocumentation("A list of merge class provide stateless component interface for merging records, and support various HoodieRecord "
```

Comment:
nit: "List of HoodieMerger implementations constituting Hudi's merging strategy -- based on the engine used Hudi will pick most efficient implementation to perform merging/combining of the records (during update, reading MOR table, etc)"